java/src/jmri/jmrix/AbstractMRTrafficController.java

```diff
@@ -595,22 +595,26 @@ public boolean hasTimeouts() {
         return timeoutFlag;
     }
 
-    private boolean timeoutFlag = false;
-    private int timeouts = 0;
+    protected boolean timeoutFlag = false;
+    protected int timeouts = 0;
```

Comment:
What's the purpose of having these protected instead of private? They doesn't seem to be used outside of this class.

Reply:
Thanks for asking. There are no getters and setters for these.  Having them `protected` lets somebody eventually override the `handleTimeout(..)` routine if needed.